### PR TITLE
WINC clarify Getting Support wording

### PR DESCRIPTION
**Pull request by mburke5678**
Per [Slack conversation](https://coreos.slack.com/archives/CM4ERHBJS/p1648832040363479?thread_ts=1648789414.418059&cid=CM4ERHBJS).

 https://docs.openshift.com/container-platform/4.9/windows_containers/windows-containers-release-notes-4-x.html#getting-support is confusing,...  I thus recommend fixing docs and pointing to https://access.redhat.com/support/policy/updates/openshift#windows ` 
 
 Previews: 
[Getting support](https://deploy-preview-44112--osdocs.netlify.app/openshift-enterprise/latest/windows_containers/windows-containers-release-notes-4-x.html#getting-support)
[OKD without the Getting support section](https://deploy-preview-44112--osdocs.netlify.app/openshift-origin/latest/windows_containers/windows-containers-release-notes-4-x).
 
 Changing 4.6 to 4.9 in separate PRs, as this section is in the release notes and only in that branch.

### CODE REVIEW

**Comment by mburke5678** [comment]
@Anandnatraj @sreber84 @mtnbikenc PTAL

File: windows_containers/windows-containers-release-notes-4-x.adoc [hunk 1]
```diff
@@ -16,7 +16,13 @@ The release notes for Red Hat OpenShift for Windows Containers tracks the develo
...
+Windows Container Support for Red Hat OpenShift is provided and available as an optional, installable component. Windows Container Support for Red Hat OpenShift is not part of the {product-title} subscription, it requires a separate subscription and is supported per the link:https://access.redhat.com/support/offerings/production/soc/[Scope of coverage] and link:https://access.redhat.com/support/offerings/production/sla[Service level agreements].
+
+You must have this separate subscription to receive support for the Red Hat WMCO. Deploying Windows container workloads in production clusters is not supported without this separate subscription. You can request support through the link:http://access.redhat.com/[Red Hat Customer Portal]. 
+
+For more information, see the Red Hat OpenShift Container Platform Life Cycle Policy document for linkhttps://access.redhat.com/support/policy/updates/openshift#windows[OpenShift Container Platform for Windows Containers].
```
Comment (mtnbikenc) [inline windows_containers/windows-containers-release-notes-4-x.adoc:23]:
```suggestion
For more information, see the Red Hat OpenShift Container Platform Life Cycle Policy document for link:https://access.redhat.com/support/policy/updates/openshift#windows[OpenShift Container Platform for Windows Containers].
```

**Review by mtnbikenc — CHANGES_REQUESTED** [review]
Looks good except there appears to be a missing colon.

**Comment by Anandnatraj** [comment]
I see mu changes above have been incorporated. Much thanks. LGTM

**Comment by mburke5678** [comment]
@rrasouli Can you PTAL? Thank you!

**Comment by mburke5678** [comment]
@rrasouli Thank you for the quick review. However, we probably should remove this statement from the OKD docs. Can you take another quick look?
https://github.com/openshift/openshift-docs/pull/44112/commits/ddc522401530cf77b15b2c09ff23bbf32f653c30

**Comment by rrasouli** [comment]
@mburke5678 hey, I can't see the diff on the preview

**Comment by mburke5678** [comment]
@rrasouli Apologies. 

Here is a preview of [OKD with the _Getting support_ section](https://deploy-preview-44112--osdocs.netlify.app/openshift-enterprise/latest/windows_containers/windows-containers-release-notes-4-x.html#getting-support)
Here is a preview of [OKD without the _Getting support_ section](https://deploy-preview-44112--osdocs.netlify.app/openshift-origin/latest/windows_containers/windows-containers-release-notes-4-x).

**Comment by mburke5678** [comment]
Closing in favor of https://github.com/openshift/openshift-docs/pull/44201

**Pull request closed without merging**